pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -481,20 +481,41 @@ class Build {
         }
         def sdkUrl = "${env.BUILD_URL}/artifact/workspace/target/${filter}/*zip*/target.zip"
         context.echo "sdkUrl is ${sdkUrl}"
-        def targets = 'sanity.jck,extended.jck,special.jck'
-        context.triggerRemoteJob abortTriggeredJob: true,
-                            blockBuildUntilComplete: false,
-                            job: 'AQA_Test_Pipeline',
-                            parameters: context.MapParameters(parameters: [context.MapParameter(name: 'SDK_RESOURCE', value: 'customized'),
-                                                                    context.MapParameter(name: 'TARGETS', value: targets),
-                                                                    context.MapParameter(name: 'CUSTOMIZED_SDK_URL', value: "${sdkUrl}"),
-                                                                    context.MapParameter(name: 'JDK_VERSIONS', value: "${jdkVersion}"),
-                                                                    context.MapParameter(name: 'PLATFORMS', value: "${platforms}")]),
-                            remoteJenkinsName: 'temurin-compliance',
-                            shouldNotFailBuild: true,
-                            token: 'RemoteTrigger',
-                            useCrumbCache: true,
-                            useJobInfoCache: true
+        def targets = ['sanity.jck', 'extended.jck', 'special.jck']
+        def parallel = 'None'
+        def num_machines = '1'
+        def remoteTargets = [:]
+
+        targets.each { target ->
+            // For each requested test, i.e 'sanity.jck', 'extended.jck', 'special.jck', call test job
+            try {
+                context.println "Remote trigger ${target}"
+                remoteTargets["${target}"] = {
+                    if ( "${target}" == 'extended.jck' && ("${platforms}" == 'x86-64_linux' || "${platforms}" == 'x86-64_windows')) {
```

Comment:
As discussed yesterday I suggest adding `x86-64_macos` as it is a primary and we can generally aim to run this over two machines. Worst case a machien will be offline, but we should aim to run on all available machines since that platform takes over 24 hours to complete.

Reply:
Enabled with x86-64_macos